manifest_staging/charts/secrets-store-csi-driver/templates/role-secretproviderclasspodstatuses-viewer.yaml

```diff
@@ -0,0 +1,24 @@
+{{ if .Values.rbac.install }}
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  labels:
+{{ include "sscd.labels" . | indent 4 }}
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
```

Comment:
aggregate to `admin` should be fine but we don't need aggregate to `edit`.

Reply:
Could you please explain why? IMO users in **either** `view`,  `edit` or `admin` role should be able to view secretproviderclasspodstatuses. Ref. https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles. The aggregation labels should be the same as the aggregation cluster role secretproviderclasses-viewer-role.

Reply:
Ah, the default aggregating user-facing roles are also aggregating each other. Makes sense - I didn't know that, thanks! Fixed now.